DataStructures/HashTable.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using LinkedList;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hash_Table
+{
+    public class HashTable : IHashTable
+    {
+        private object[] _array;
+        public int Length => _array.Length;
+
+
+        public HashTable()
+        {
+            _array = new object[3];
```

Comment:
3 is a magic number. Avoid this when you can. Instead, you can create a constant variable and give it a name DefaultSize or something like that

Reply:
Okay